### PR TITLE
Feature/tawilind

### PR DESCRIPTION
**Pull request by leifermendez**
# What is it?
New command for add Tailwind in your Qwik Ecosystem
- [x ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

New feature for add Tailwind easy to Qwik App. Just need run  `npm run qwik add tailwind`
image

image

# Checklist:

- [ x] My code follows the [developer guidelines of this project](../CONTRIBUTING.md)
- [x ] I have performed a self-review of my own code
- [xx ] I have made corresponding changes to the documentation
- [ x] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by leifermendez** [comment]
@manucorporat 
Could you check the next file `tailwind.js`  it file add a few lines inside `global.css `. This script is execute postinstall and selftdelete it (😅)
https://github.com/BuilderIO/qwik/blob/79f0f87720cc8157ccee76ccdf99febe2b83484b/starters/features/tailwind/tailwind.js

File: packages/qwik/src/cli/types.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export interface UpdateAppOptions {
   rootDir: string;
   integration: string;
   installDeps?: boolean;
+  postInstallScript?: boolean;
```
Comment (leifermendez) [inline packages/qwik/src/cli/types.ts:14]: I need remove it.

**Comment by leifermendez** [comment]
i switch to draft for re-check

**Comment by manucorporat** [comment]
This is amazing.... i am not sure about the postscript check... need to check with @adamdbradley . I think we should have a good system to print next step after the install, i image we could do something with:

**ACTION REQUIRED**: Add the follow lines to your `global.css` file:

```diff
+ @tailwind base;
+ @tailwind components;
+ @tailwind utilities;
```

**Comment by leifermendez** [comment]
> This is amazing.... i am not sure about the postscript check... need to check with @adamdbradley . I think we should have a good system to print next step after the install, i image we could do something with:
> 
> **ACTION REQUIRED**: Add the follow lines to your `global.css` file:
> 
> ```diff
> + @tailwind base;
> + @tailwind components;
> + @tailwind utilities;
> ```

Ok perfect! I will work on this part for clean unnecessary files ✌️

**Comment by leifermendez** [comment]
https://www.loom.com/share/c978ee42cdf044d9ad682ad08621b53d

File: starters/features/tailwind/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Use Tailwind into your Qwik app",
```
Comment (ShaggyDude) [inline starters/features/tailwind/package.json:2]: "description": "Use Tailwind inside your Qwik app",
Seems a little more accurate?

**Comment by ShaggyDude** [comment]
Very much a side note but if you use VS code install this => https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss

**Comment by manucorporat** [comment]
I see the PR also includes changes in partytown, is this expected?

**Comment by leifermendez** [comment]
@manucorporat 

> I see the PR also includes changes in partytown, is this expected?
No, I will remove it. I don't know wait a moment

**Comment by leifermendez** [comment]
@manucorporat I don't know how now one check is fail

**Comment by leifermendez** [comment]
> @manucorporat I don't know how now one check is fail

I get it, I see the origin of the fail check is line 86 but this file I don't touch maybe is a new implementation 
  https://github.com/BuilderIO/qwik/blob/3ac65cb2f49091ec3d756bd17a202f747950eb48/packages/qwik/src/qwikloader.ts#L86

**Pull request merged**